guide/english/wordpress/index.md

```diff
@@ -44,12 +44,12 @@ It supports many useful publishing features out of the box, like the ability to
 
 In addition to posts, WordPress also comes with the ability to create "pages" — any non-chronological page on your site, like an "About" or "Contact" page. 
 
-What makes WordPress particularly powerful is that it's possible to create other custom page  types as well. You could, for example, create your own classifieds site with a "listings" page type, or a book review site with a "review" page type, and even assign custom data fields and taxonomies to these as well. Using a combination of plugins and custom coding, you can adopt WordPress as a content management system for all kinds of uses.
+What makes WordPress particularly powerful is that it's possible to create other custom page types as well. You could, for example, create your own classifieds site with a "listings" page type, or a book review site with a "review" page type, and even assign custom data fields and taxonomies to these as well. Using a combination of plugins and custom coding, you can adopt WordPress as a content management system for all kinds of uses.
 
 ### Templates
 
-WordPress does not require work from scratch. For ease of new developers there are tons of free templates available on internet which can be used by basic developer. 
-Few websites which provides these free templates are:
+WordPress does not require work from scratch. For ease of new developers, there are tons of free templates available on the internet which can be used by a basic developer. 
+Few websites which provide these free templates are:
```

Comment:
`which provide these` can be termed better as `which are providing these`, please make that required change